data_tamer_cpp/CMakeLists.txt

```diff
@@ -16,17 +16,21 @@ endif()
 
 option(BUILD_SHARED_LIBS "Build using shared libraries" OFF)
 
-find_package(ament_cmake QUIET)
-
-if (ament_cmake_FOUND)
-    set(COMPILING_FOR_ROS2 ON)
+# optionally build for ROS 2
+option(BUILD_ROS "Build for ROS 2" ON)
```

Comment:
- DATA_TAMER_BUILD_ROS would be more consistent and would help to avoid possible naming collisions
- the variable should be checked in `examples/CMakeLists.txt` as well